pipelines/jobs/configurations/jdk11u_pipeline_config.groovy

```diff
@@ -174,6 +204,9 @@ class Config11 {
                 test                : [
                         nightly: ['sanity.openjdk'],
                         weekly : ['sanity.openjdk', 'sanity.system', 'extended.system', 'sanity.perf']
+                ],
+                buildArgs           : [
```

Comment:
Ther is already a buildArgs section for other variants in this section - please unify them, otherwise it will lead to confusion at best, and at worst not actually work.